### ISSUE TITLE
Polls via Forms on Nostr. 

### ISSUE THREAD

**Issue by abh3po** [nostrworld/nostriga#35]
# Description
### What is this talk about? Give us as many details as possible. 
In this talk I'll try to go over what are the essential properties of a polling system and how we can use nostr to create a transparent polling mechanism that can work beyond social media polling by incorporating https://github.com/nostr-protocol/nips/pull/1190 . I'll talk about the attempts that have been made to solve this problem previously and why this approach solves the problems of the previous attempts as well as the tradeoffs that are taken in this method as well.
### What would an attendee learn from this talk?
The attendee will learn about the various challenges that we face while creating a voting system on nostr. What were the previous attempts made to solve this, how anyone can create clients for polls and forms on nostr. 
### Is there anything folks should read up on before they attend this talk?
Going through the nips for zap polls, and forms would be good for added context.
### Relevant Links
[Zap polls](https://github.com/nostr-protocol/nips/pull/320)
[forms](https://github.com/nostr-protocol/nips/pull/1190)
[polls](https://github.com/nostr-protocol/nips/pull/1346)
# About the Speaker
I am the author of https://formstr.app , and I've been working nostr since September 2023.  
### Social Links
Nostr (https://njump.me/abhay@formstr.app)
Github (https://github.com/abhay-raizada)

# Talk Details
### Length of Talk
25 mins
### Preferred Day/Time Slot 
Anytime

**Comment by exfrog** [comment]
LGTM

**Comment by sarahiscoding** [comment]
Super cute start. Just tested it out, made a breakfast poll, and received a couple of responses: cheezburger, coffee and biscuits. 

https://formstr.app/#/fill/98f7c35a88d0bbc14ceb68825b988126c8af5c2ae072e12eea29d7ee65241c3c

https://formstr.app/#/response/f6d99fa530ead6bd6ad4381302712f7fedf39450feb1a5e199c8e131c9cd4464

I like the idea of hearing about Nostr polling, possibly a panel from different viewpoints, the pros and cons, what's been working vs what could be improved, etc. 

To sum up: could this be a Polling Panel?

**Comment by abh3po** [comment]
> To sum up: could this be a Polling Panel?

could make it a polling panel. 

Though I also dropped a separate NIP for polls: https://github.com/nostr-protocol/nips/pull/1346 and a 
polling client as well https://pollerama.fun 
which I was planning on covering in the talk.

**Comment by abh3po** [comment]
Hey closing this, since I won't be able to join in Riga (issues with travel), would've loved to be there :/ , maybe next time
